config/FCCAnalysisRun.py

```diff
@@ -405,6 +408,7 @@ def runLocal(rdfModule, fileList, output, batch):
     publicOptions = parser.add_argument_group('User options')
     publicOptions.add_argument("--files-list", help="Specify input file to bypass the processList", default=[], nargs='+')
     publicOptions.add_argument("--output", help="Specify ouput file name to bypass the processList and or outputList, default output.root", type=str, default="output.root")
+    publicOptions.add_argument("--test", action='store_true', help="Run over the test file", default=False)
```

Comment:
```suggestion
    publicOptions.add_argument("--testFile", action='store_true', help="Run over the test file", default=False)
```

or rename to `args.test` below?

Reply:
probably better to rename below